extensions/amp-fit-text/0.1/amp-fit-text.js

```diff
@@ -118,11 +122,37 @@ class AmpFitText extends AMP.BaseElement {
 
   /** @override */
   layoutCallback() {
+    if (this.win.ResizeObserver && this.resizeObserverUnlistener_ === null) {
+      const observer = new this.win.ResizeObserver(() =>
+        throttle(
```

Comment:
This is not how `throttle` method works. 
If you look at the implementation, `throttle` doesn't invoke the callback, it only wraps the given callback. 
I wonder why the test can pass.

Reply:
Ah I see, I've modified this to wrap the callback entirely.